security/imx6/gen_close.sh

```diff
@@ -86,7 +86,7 @@ do
 done
 echo
 cat << EOF
-FB: ucmd if fiohab_close; then echo Platform Secured; else echo Error, Can Not Secure the Platform; fi
+FB[-t 1000]: ucmd if fiohab_close; then echo Platform Secured; else echo Error, Can Not Secure the Platform;sleep 2; fi
```

Comment:
Mind just adding a space between ';' and sleep 2? Same for the other files.

Reply:
Done